devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -41,6 +41,9 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05 ,1000, FEDC", outClusters: "000A, 0019", manufacturer: "Aurora", model: "FWG125Bulb50AU", deviceJoinName: "Aurora Dimmable Filament Vintage G125"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05 ,1000, FEDC", outClusters: "000A, 0019", manufacturer: "Aurora", model: "FWA60Bulb50AU", deviceJoinName: "Aurora Dimmable Filament Vintage GLS"
 
+		// IKEA
+		fingerprint manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E27 WW 806lm", deviceJoinName: "Ikea TRÅDFRI LED bulb E27 806 lumen, wireless dimmable, warm white opal white" // raw description 01 0104 0101 01 07 0000 0003 0004 0005 0006 0008 1000 04 0005 0019 0020 1000
```

Comment:
Most of the Ikea fingerprints have a `deviceJoinName` of "IKEA TRÅDFRI LED Bulb". For consistency sake (and to keep it short) I think we should use that here too. Thoughts?

Reply:
I took this name from development task. I don't mind to use short version.

Reply:
I suggest noting in the exception in the ticket but I think it's better with the shorter version